common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -97,6 +97,8 @@ public enum Feature {
         DEVICE_FLOW("OAuth 2.0 Device Authorization Grant", Type.DEFAULT),
 
         TRANSIENT_USERS("Transient users for brokering", Type.EXPERIMENTAL),
+
+        MULTI_SITE("Multiple sites support", Type.PREVIEW),
```

Comment:
I'm not sure about the name. The load balancer checker is useful outside of cross-dc support; for example, if a user configured the embedded Infinispan with a database (JDBC cache store). Probably some name related to persistent sessions would be a better fit?

Reply:
I agree this check `lb-check` has the potential to be useful in other places as well. For now, I'd like to narrow its use down to multi-site as this gives us the chance to modify it as needed which the check we want to include. I was even considering to name the URL ` lb-check-for-multi-site`, but let's be a little bit ambitious here that it might be useful in other scenarios as well  :-)